### PR TITLE
Azure log mapping solution

### PR DESCRIPTION
**Pull request by vishaalcharpe**
Added a new field AzureAccountName to template.

### CODE REVIEW

File: arm-template-deployment/deployRGParent.json [hunk 1]
```diff
@@ -45,6 +45,12 @@
         "description": "Application Client Id provided during the creation of azure account in LM"
       }
     },
+    "Azure_Account_Name": {
```
Comment (choudharypooja) [inline arm-template-deployment/deployRGParent.json:48]: Can we make this parameter optional as the user can map to existing clientId if not the azure account?

**Comment by vishaalcharpe** [comment]
Updated the Azure account name as optional field. Also, optimised the null check on the same field.

**Review by choudharypooja — APPROVED** [review]
LGTM

**Review by siddharthck — APPROVED** [review]
LGTM

**Pull request merged**